environment_oscar.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
   - cpuonly=1.0=0
```

Comment:
Do we feel like this yml file is needed in light of the smaller and newer Marijuana_Study.yml? I would recommend making needed adjustments to that and saying goodbye to this one because of the unwieldy version constraints

Reply:
Agreed, removing this.